<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MazeView</class>
 <widget class="QMainWindow" name="MazeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MazeView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(0,0,0);
}

QPushButton {
	background-color: white;
	color: black;
}
QLabel {
	color: white;
}
QSpinBox {
	color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonSave">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>460</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>SaveFile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOpen">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>400</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: white;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>OpenFile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGenerate">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRows">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ROWS:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxRows">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelCols">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>COLS:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCols">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>260</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>504</width>
      <height>504</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView {
border: 2px solid white ;
}</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
